test/clt-tests/core/generate-1mln-records.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 set -e
 
-total_iterations=20000
```

Comment:
I think there is no point to remove it from here. Let make fallback

`total_iterations=${total_iterations:-2000}`

Reply:
I removed it from the inside so I could change it from the outside via an argument